modules/gridBidAdapter.js

```diff
@@ -167,6 +176,11 @@ export const spec = {
       request.site.content = content;
     }
 
+    const ortb2SiteContent = deepAccess(bidOrtb2, 'site.content');
+    if (ortb2SiteContent) {
+      request.site.content = mergeDeep(request.site.content || {}, ortb2SiteContent);
```

Comment:
If you get all of it, can you simplify other places in your adapter, like where you get request.site.content.genre specifically?

Reply:
Hello! But here we get site.content from bid.ortb2, and site.content.genre we get from config

Reply:
but those are the same?

Reply:
Sorry, can you provide a bit more details about the source of your concern here, not quite following